Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -19,6 +19,8 @@ private let CancelString = NSLocalizedString("Cancel", comment: "Cancel button")
 private let KVOLoading = "loading"
 private let KVOEstimatedProgress = "estimatedProgress"
 
+private let maxDialogUrlLength = 225
```

Comment:
Ahh yes, the search and replace function. Sorry I didn't realize that was the context for that line. Gee, I really wish Xcode had a vim mode for the text editor. Would make navigating these long files so much easier